pytest_bdd/scenario.py

```diff
@@ -111,8 +110,12 @@ def _execute_step_function(request, scenario, step, step_func):
         kw["step_func_args"] = kwargs
 
         request.config.hook.pytest_bdd_before_step_call(**kw)
+        target_fixture = getattr(step_func, "target_fixture", None)
```

Comment:
is this going to allow "when" and "then" steps to have a target_fixture param?

Reply:
Only in case if the signature allows it. Currently not the case